src/multi-selector.css

```diff
@@ -2,13 +2,14 @@
 	display: inline-block;
 	position: relative;
 	font-size: 1.4rem;
-}
-
-.cpr-multi-selector {
 	width: 340px;
 	z-index: 1;
 }
 
+.cpr-multi-selector--active {
+	z-index: 8;
```

Comment:
since the dialog for the pillbox is inside of the pillbox, it cannot have a greater z-index than its container. When you have two pillboxes (one open and one closed), this is a problem.